### PR TITLE
Scaffold new app for VA Form 26-4555

### PR DESCRIPTION
**Pull request by tlei123**
## Summary

- Scaffold new vets-website app for **VA Form 26-4555 &mdash; Application in Acquiring Specially Adapted Housing or Special Home Adaptation Grant**
- **VA.gov Product Forms** team initiative
- [no feature-flag just yet; content-build apps-registry entry has `vagovprod` property set to `false`]

## Related PR
[content-build#1454](https://github.com/department-of-veterans-affairs/content-build/pull/1454)

## Related issue(s)
- department-of-veterans-affairs/va.gov-team-forms#14
- [Epic](https://github.com/department-of-veterans-affairs/va.gov-team-forms/issues/28)

## Testing done

[none yet; this is just the preliminary form-config for now]

## What areas of the site does it impact?
[none yet; no content-page or search-tool currently refers/links to this new form]

## Acceptance criteria

- [ ]  New form-config/-code approved

## Requested Feedback

Just the regular code-review of my newly scaffolded form.

### CODE REVIEW

File: src/hello-world/hello.txt
Comment (gmrabian) [inline src/hello-world/hello.txt:6]: I'm guessing we don't want this to be in the PR 😅

File: .yarnrc
Comment (gmrabian) [inline .yarnrc:7]: are these changes, and the .yarn/releases changes meant to be in the form PR? It's massive.

File: src/applications/simple-forms/26-4555/containers/ConfirmationPage.jsx
Comment (tlei123) [reply src/applications/simple-forms/26-4555/containers/ConfirmationPage.jsx:1]: This eslint-disable will be deleted as part of final changes.  Just preserving some auto-generated stuff for now in case it's needed later.

File: src/applications/simple-forms/26-4555/containers/ConfirmationPage.jsx
Comment (tlei123) [reply src/applications/simple-forms/26-4555/containers/ConfirmationPage.jsx:2]: This eslint-disable will be deleted as part of final changes. Just preserving some auto-generated stuff for now in case it's needed later.

File: src/applications/simple-forms/26-4555/pages/serviceHistory.js
Comment (tlei123) [reply src/applications/simple-forms/26-4555/pages/serviceHistory.js:1]: This eslint-disable will be deleted as part of final changes. Just preserving some auto-generated stuff for now in case it's needed later.

File: src/hello-world/hello.txt
Comment (tlei123) [reply src/hello-world/hello.txt]: Deleted.

File: .yarnrc
Comment (tlei123) [reply .yarnrc]: Reverted.

**Comment by tlei123** [comment]
Unit-test checks finally passed, after a few days of main-branch merges.  Not sure about what to do with the other failed checks.  Hoping those won't block the merge once PR's approved.

**Comment by tlei123** [comment]
Now a couple of E2E checks are failing during the Install dependencies action-steps.  Not sure what to do with these failures either &mdash; my one single E2E-spec for this new form is [skipped in CI.](https://github.com/department-of-veterans-affairs/vets-website/blob/0793458f70cda90561737b6f6ac7facb80d92c03/src/applications/simple-forms/26-4555/tests/4555-adapted-housing.cypress.spec.js#L37)

**Review by rmessina1010 — APPROVED** [review]
Approved

**Pull request merged**
